api-call.py

```diff
@@ -1,29 +1,226 @@
+
+
+def get_user_choice():
+    """Get user input from menu
+
+    :returns: The user choice
+    """
+
+    config = load_from_config()
+    return input(config['generalStrings']['enterChoice'])
+
+
+def get_url(choice):
+    """Get URL for API
+
+    :param choice: user choice input
+    :returns: API URL Object after user choice
+    """
+
+    config = load_from_config()
+    access_token = get_access_token()
+    content_type = config['accessToken']['headers']['Content-Type']
+    header = {'Content-Type': content_type, 'Authorization': access_token}
+    url_obj = {}
+    if choice == '0':
+        url_obj = {'url': config['generalStrings']['exiting'],
+                   'parameters': {},
+                   'header': header}
+    elif choice == '1':
+        url_obj = {'url': config['apiUrls']['beaverBus'],
+                   'parameters': {},
+                   'header': header}
+    elif choice == '2':
+        url_obj = {'url': config['apiUrls']['terms'],
+                   'parameters': {},
+                   'header': header}
+    elif choice == '3':
+        user_date = input('Enter Date (yyyy-mm-dd): ')
+        url_obj = {'url_terms': config['apiUrls']['terms'],
+                   'url_textbooks': config['apiUrls']['textbooks'],
+                   'parameters': {'date': user_date},
+                   'header': header}
+    elif choice == '4':
+        route_id = input('Enter Route Number: ')
+        url_obj = {'url_routes': config['apiUrls']['routes'],
+                   'url_arrivals': config['apiUrls']['arrivals'],
+                   'url_vehicles': config['apiUrls']['vehicles'],
+                   'route_id': route_id,  'parameters': {},
+                   'header': header}
+    else:
+        print(config['generalStrings']['invalidChoice'])
+
+    return url_obj
+
+
+def get_text_books_with_term_date(url_obj):
+    """Get text books based on specified term and date
+    """
+
+    terms_data_response = get_api_data(url_obj['url_terms'],
+                                       url_obj['parameters'],
+                                       url_obj['header'])
+    academic_year = terms_data_response['data'][0]['attributes']['calendarYear']
+    term = terms_data_response['data'][0]['attributes']['season']
+    url_obj['textbooks_parameters'] = {'academicYear': academic_year,
+                                       'term': term,
+                                       'subject': 'CS',
+                                       'courseNumber': '161'}
+
+
+def get_stops_vehicles_on_route(url_obj):
+    """Get stops and vehicles on a given route
+    and determine where it is heading in real time
+    and the ETA for the stop
+    """
+
+    routes_data_response = get_api_data(url_obj['url_routes']+'/'+url_obj['route_id'],
+                                        url_obj['parameters'],
+                                        url_obj['header'])
+    route_name = routes_data_response['data']["attributes"]["description"]
```

Comment:
Be careful of the use of single-quotes and double-quotes. Let's make them consistent and please also check your entire script.

Reply:
done.